static/css/_landing.scss

```diff
@@ -0,0 +1,637 @@
+@import url('https://fonts.googleapis.com/css2?family=IBM+Plex+Sans&family=Inconsolata&family=Montserrat:ital,wght@0,900;1,700&family=Poppins&family=Sura&display=swap');
+
+$font-IBM: 'IBM Plex Sans',
+sans-serif;
+
+.l-container {
+  padding: 0 2rem;
+
+  @media (min-width: 1440px) {
+    max-width: 1400px;
+  }
+}
+
+.hero {
+  background-color: #08223d;
+  background-image: url(../image/BG-1.webp);
+  background-repeat: no-repeat;
+  background-size: cover;
+
+  &-1 {
+    display: flex;
+    align-items: center;
+    margin: 0 auto;
+    padding: calc(7rem + 5rem) 0px;
+
+    @media screen and (max-width: 1440px) {
+      padding: calc(7rem + 5rem) 2rem;
+    }
+
+    @media screen and (max-width: 991px) {
+      flex-direction: column-reverse;
+      padding: 4rem 2rem;
+
+      .btn-group {
+        text-align: left;
+      }
+    }
+
+    &__content {
+      flex: 1 1 50%;
+
+      h1 {
+        color: #FFF;
+        margin: 2rem 2rem 2rem 0;
+        font: 700 5rem $font-IBM;
+
+        @media screen and (max-width: 991px) {
+          font: 700 3rem "IBM Plex Sans", sans-serif;
+          margin: 0 0 2rem 0;
+        }
+      }
+
+      h2 {
+        color: #fff;
+        line-height: 40px;
+        margin-bottom: 2rem;
+        font: 400 2rem $font-IBM;
+
+        @media screen and (max-width: 991px) {
+          font: 400 1.5rem "IBM Plex Sans", sans-serif;
+          margin: 0 0 2.5rem 0;
+        }
+      }
+    }
+
+    &__placeholder {
+      flex: 1 1 50%;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+
+      @media screen and (max-width: 991px) {
+        flex-direction: column;
+      }
+
+      img {
+        @media screen and (max-width: 991px) {
+          object-fit: contain;
+          width: 100%;
+        }
+      }
+    }
+
+    &__binary_logo {
+      display: none;
+      width: 14.75rem !important;
+      margin-bottom: 3rem;
+
+      @media screen and (max-width: 991px) {
+        display: block;
+      }
+    }
+  }
+
+
+
+}
+
+.l-btn {
+  height: 2.5rem;
+  border-radius: 0.25rem;
+  font: 700 0.875rem $font-IBM;
+  background-color: #C2C2C2;
```

Comment:
Can we use variables? I think we already have some, please check if we don't have these added to the variables.